guest-test/tdx_osv_sanity/tdx_osv_sanity.test_executor.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-2.0-only
+# Copyright (c) 2023 Intel Corporation
+
+# Author:   Hongyu Ning <hongyu.ning@intel.com>
+#
+# History:  5, Dec., 2023 - Hongyu Ning - creation
+
+
+# @desc This script prepare and run $TESTCASE in for $FEATURE tdx_osv_sanity
+# in tdx VM; please note, $FEATURE name and subfolder name under guest-test
+# must be the exactly same, here FEATURE=tdx_osv_sanity
+
+###################### Variables ######################
+## common variables example ##
+SCRIPT_DIR_LOCAL="$( cd "$( dirname "$0" )" && pwd )"
+echo "$SCRIPT_DIR_LOCAL"
+# get test scenario config for $FEATURE tdx test_executor
+source "$SCRIPT_DIR_LOCAL"/../test_params.py
+## end of common variables example ##
+
+###################### Functions ######################
+## $FEATURE specific Functions ##
+
+###################### Do Works ######################
+## common works example ## 
+cd "$(dirname "$0")" 2>/dev/null || exit 1
```

Comment:
That's my concern about the quote marks, what do you really want to quote? Here are "$(dirname " and ")". Are they correct?